help/de/Help_ch_Account-Actions.xml

```diff
@@ -514,92 +523,142 @@
+          </listitem>
+        </varlistentry>
 
-        <listitem>
-          <para><guilabel>Kleinste Stückelung:</guilabel> Die kleinste Teilung, die erfasst werden kann.
-          </para>
-        </listitem>
+        <varlistentry>
+          <term><guilabel>Wertpapier/Währung</guilabel></term>
 
-        <listitem>
-          <para><guilabel>Kontofarbe:</guilabel> Die Farbe, die der Registerkarte des Kontos zugewiesen werden soll.
-            Klicken Sie auf die Schaltfläche <guibutton>Farbe</guibutton>, um die Anzeige
-            <guilabel>Farbe auswählen</guilabel> zu öffnen. Wollen Sie die Farbeinstellung für
-            das Kontos zurückzusetzen betätigen Sie die Schaltfläche
-            <guibutton>Voreinstellung</guibutton>.
-          </para>
-        </listitem>
+          <listitem>
+            <note>
+              <para>Der <guilabel>Typ</guilabel> im Feld des <guilabel>Wertpapier auswählen</guilabel>-Dialogs ist
+                abhängig von der Auswahl der <guilabel>Kontoart</guilabel> in der <guilabel>Neues
+                Konto</guilabel> Anzeige.
+              </para>
+            </note>
 
-        <listitem>
-          <para><guilabel>Bemerkung:</guilabel> Dies ist ein Freitextfeld, das für zusätzliche Notizen über das
-            Konto verwendet werden kann.
-          </para>
-        </listitem>
-      </orderedlist>
+            <itemizedlist spacing="compact">
+              <listitem>
+                <para>Für die meisten Konten wird hier die Standardwährung, d.h. Ihre Landeswährung, z.B. <quote>EUR
+                  (Euro)</quote> angezeigt. Falls das Konto für eine Fremdwährung vorgesehen ist,
+                  benutzen Sie die <guibutton>Auswählen&hellip;</guibutton> Schaltfläche und
+                  wählen im folgenden Auswahldialog eine andere Währung.
+                </para>
+              </listitem>
+
+              <listitem>
+                <para>Für Konten, die <emphasis>Wertpapiere</emphasis> enthalten, wählen Sie zunächst
+                  <guilabel>Aktie</guilabel> oder <guilabel>Investmentfonds</guilabel> in dem Feld
+                  <guilabel>Kontoart</guilabel> aus, dann das <guilabel>Hauptkonto</guilabel> und im
+                  Anschluss benutzen Sie die Schaltfläche
+                  <guibutton>Auswählen&hellip;</guibutton>, um den <guilabel>Typ</guilabel>
+                  auszuwählen sowie das Wertpapier im <guilabel>Wertpapier
+                  wählen</guilabel>-Dialog.
+                </para>
+<!-- FIXME source: new note -->
+                <note>
+                  <para>Ähnlich der Kontonummer bietet der Typ, der intern Namensraum heißt, eine Möglichkeit, seine
+                    Investments sachlich zu sortieren. Grundsätzlich bietet es sich an, sie nach
+                    Risiko zu gruppieren. Zum Beispiel kann als erste Ebene der Typ des Wertpapiers
+                    angegeben werden und als optionale 2. Ebene die Nominalwährung.
+<screen>
+:
```

Comment:
Ich würde den Doppelpunkt und Screen –auch unten– in eine Zeile setzen, sonst gibt es überflüssige Leerzeilen.

Reply:
ok.